data/os/RedHat.yaml

```diff
@@ -1,26 +1,26 @@
 ---
 profile_firewall::inbuilt_chains:
```

Comment:
I audited and fixed the ordering of these. Nothing was technically wrong with the ordering, but I sorted each group of tables alphabetically.